src/mango/src/mango_cursor_view.erl

```diff
@@ -124,25 +128,31 @@ composite_prefix([Col | Rest], Ranges) ->
             []
     end.
 
-
-% Low and behold our query planner. Or something.
-% So stupid, but we can fix this up later. First
-% pass: Sort the IndexRanges by (num_columns, idx_name)
-% and return the first element. Yes. Its going to
-% be that dumb for now.
+% Low and behold our query planner
+% First pass: sort the IndexRanges by KeysPrefixLength,
```

Comment:
"First pass:" is kind of odd since we're not labeling anything else. I would rephrase this sentence to avoid the colon there.